e2e-tests/server/app.py

```diff
@@ -18,6 +18,6 @@ def create_app():
     app.add_route('/sse', Events(hub))
     app.add_route('/ws/{name}', Chat(hub))
 
-    app.add_static_route('/static', str(STATIC))
+    app.add_static_route('/static', STATIC)
```

Comment:
Changed this to serve as an extra validation of https://github.com/falconry/falcon/issues/1962.